tests/python-gpu/conftest.py

```diff
@@ -0,0 +1,40 @@
+import pytest
```

Comment:
@RAMitchell We should export our compilation info to Python in the future, including whether XGBoost has CUDA, NCCL, RMM etc.